cyclonedx-bom/src/specs/v1_3/dependency.rs

```diff
@@ -83,14 +83,14 @@ impl FromXml for Dependencies {
 pub(crate) struct Dependency {
     #[serde(rename = "ref")]
     dependency_ref: String,
-    depends_on: Vec<String>,
+    depends_on: Option<Vec<String>>,
```

Comment:
If you annotate the `depends_on` field with [serde's `default` attribute](https://serde.rs/attr-default.html)
```suggestion
    #[serde(default)]
    depends_on: Vec<String>,
```

an empty `Vec` (default) is set instead in case the field is absent from the JSON input. Then all other changes can be reverted.

Reply:
This is for version 1.3. Would it be ok to apply the change to the 1.4 version of [Dependency](https://github.com/CycloneDX/cyclonedx-rust-cargo/blob/main/cyclonedx-bom/src/specs/v1_4/dependency.rs#L86) too as the behavior according to the [Spec 1.4](https://cyclonedx.org/docs/1.4/json/#dependencies_items_dependsOn) is the same?

Reply:
Thanks for your review @justahero ! I've applied your suggested change.